### ISSUE TITLE
Creting form via no-code builder results in 404: /forms/undefined/form

### ISSUE THREAD

**Issue by tylerssn** [formbricks/formbricks#66]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

<img width="737" alt="image" src="https://user-images.githubusercontent.com/4887071/185707936-f80e5e3a-201e-46df-a20e-3daa172b1e73.png">

<img width="958" alt="image" src="https://user-images.githubusercontent.com/4887071/185707991-efb96eb8-72b7-44e7-89f6-7267e0ed49bb.png">

### Steps to Reproduce

(Write your answer here.)

## Environment

- [ ] snoopForms Cloud (app.snoopforms.com)
- [x] self-hosted snoopForms, version/commit: [please provide]

**Comment by bytrangle** [comment]
I'm unable to recreate this. It took quite a while to load the code builder, but it still worked fine for me.

**Comment by svr8** [comment]
Facing the same issue here

**Comment by ohemelaar** [comment]
I faced the same issue while trying to deploy it locally. The problem was a misconfigured `NEXTAUTH_URL`. I wanted a "real" domain name for snoopForms while testing so I edited `/etc/hosts` to point `snoopforms.local` to my minikube VM and set `NEXTAUTH_URL` to `https://snoopforms.local` but this actually doesn't work. I think it's because something in snoopforms actually tries to make the request to a host that it doesn't know as the `/etc/hosts` hack only has effect outside containers. Setting it back to `http://localhost:3000` made it work again.
